javascript/ql/src/semmle/javascript/frameworks/SystemCommandExecutors.qll

```diff
@@ -7,15 +7,17 @@ import javascript
 
 private class SystemCommandExecutors extends SystemCommandExecution, DataFlow::InvokeNode {
   int cmdArg;
+  int optionsArg;
```

Comment:
Please explain the meaning of `optionsArg = -1` and `optionsArg = -2` in a comment here.